src/ownership/move-semantics.md

```diff
@@ -21,6 +20,10 @@ fn main() {
 
 * Mention that this is the opposite of the defaults in C++, which copies by value unless you use `std::move` (and the move constructor is defined!).
 
+* Data is not moved when you do `s2 = s1`, it is only the ownership that moves.
```

Comment:
"Data .. moved" I think is referring to copy bytes here? I think this is a confusing way to use the word "move". And also, whether bytes get copied or not is actually about optimization, which is not what we're talking about here. Maybe change the emphasis:
```suggestion
* It is only the ownership that moves. Whether the resulting machine code copies any data around is a matter of optimization, and such copies are aggressively optimized away.
```